src/wp-admin/css/media.css

```diff
@@ -974,7 +974,7 @@ border color while dragging a file over the uploader drop area */
 .image-editor .imgedit-menu .button {
 	display: inline-block;
 	width: auto;
-	min-height: 28px;
+	min-height: 30px;
 	font-size: 13px;
 	line-height: 2;
```

Comment:
```suggestion
	line-height: 2.15384615; /* 28px */
```

The image editing buttons probably should change the `line-height` too. r46866 updated both the `line-height` and `max-height` for `.wp-core-ui .button` and more elements. (The other two pixels are the top and bottom border.)